browses.md

```diff
@@ -54,6 +54,27 @@ Error codes:
 
 **404** if the browse index doesn't exist
 
+## Browse indexes configured for metadata linking
+### Get a single index configured to render given field as a link
+**/api/discover/browses/search/byField?field=<:field>**
+
+Example: <http://dspace7.4science.it/dspace-spring-rest/#http://dspace7.4science.it/dspace-spring-rest/api/discover/browses/search/byField/dc.contributor.author>
```

Comment:
can you please update the url to the current demo?

Reply:
I've updated this for the (only remaining) endpoint byFields, and fixed the doc / demo example